### PR TITLE
LTD-5440 fix current dependabot alerts

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

Pipfile and package.json updated as per [here](https://github.com/uktrade/lite-frontend/security/dependabot)

Deployed to dev2 environment.

[LTD-5440](https://uktrade.atlassian.net/browse/LTD-5440)

[LTD-5440]: https://uktrade.atlassian.net/browse/LTD-5440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by hnryjmes — COMMENTED** [review]
has this branch been deployed to an environment to check the dependencies build properly?

File: Pipfile [hunk 1]
```diff
@@ -52,18 +52,18 @@ markdown = "~=3.4.1"
 mohawk = ">=1.1.0"
 munch = "~=2.5.0"
 requests = {extras = ["security"], version = "~=2.21"}
-sentry-sdk = "~=1.17.0"
+sentry-sdk = "~=2.8.0"
```
Comment (currycoder) [inline Pipfile:55]: Can we keep the major version for this one on 1.X for now?  Or do we *have* to bump this to 2.X to make the alert go away?  Reason I ask is that I am fairly sure that the bump for this from 1.X to 2.X on lite-hmrc is the cause of our `LookupError` noise in lite-internal-alerts and I'm keen not to add even more noise.  At least until we solve that for lite-hmrc.

Comment (markj0hnst0n) [reply Pipfile:55]: That version is the minimum required to make the alert go away, yeah.

Comment (markj0hnst0n) [reply Pipfile:55]: Do you want me to remove this update?

Comment (currycoder) [reply Pipfile:55]: No it's ok we can keep it in.  Seems like lite-api already has a 2.X version and isn't spewing errors so hopefully the issue I'm talking about won't occur here either.

**Comment by markj0hnst0n** [comment]
@hnryjmes I've deployed this to dev2 now and am making some checks.  Looks okay so far.

**Review by depsiatwal — APPROVED** [review]
lgtm

**Pull request merged**